package.json

```diff
@@ -16,6 +16,8 @@
     "test": "vitest run",
     "test:watch": "vitest",
     "test:coverage": "vitest run --coverage",
+    "test:storybook": "test-storybook",
```

Comment:
Could you add this to the CI workflow?

Reply:
I'll try 😉

Reply:
I think you have to give third party PRs permission to run actions. If you see the **checks**, none of the workflows are running.